src/bin/vip-backup.ts

```diff
@@ -4,10 +4,14 @@ import command from '../lib/cli/command';
 import { trackEvent } from '../lib/tracker';
 
 void command( { usage: 'vip backup' } )
-	.command( 'db', 'Trigger a new backup for your database' )
+	.command( 'db', 'Generate a fresh database backup for an environment.' )
```

Comment:
I'm not *sure* this is better. Just a thought.

```suggestion
	.command( 'db', 'Generate a fresh database backup of an environment.' )
```

Reply:
+1 - I agree with the preference to use "of" here like we [applied to the other command](https://github.com/Automattic/vip-cli/pull/1766#discussion_r1559750114).